lib/postgrex/extensions/binary.ex

```diff
@@ -162,10 +162,15 @@ defmodule Postgrex.Extensions.Binary do
       bin = for digit <- digits, into: "", do: <<digit :: uint16>>
       ndigits = div(byte_size(bin), 2)
 
-      [<<ndigits :: int16, weight :: int16, sign :: uint16, scale :: int16>>, bin]
```

Comment:
Just to keep the same style as [here](https://github.com/ericmj/postgrex/blob/master/lib/postgrex/extensions/binary.ex#L138), it was not really necessary. :)